pkg/machinepool/output.go

```diff
@@ -27,6 +27,7 @@ var nodePoolOutputString string = "\n" +
 	"Kubelet configs:                       %s\n" +
 	"Additional security group IDs:         %s\n" +
 	"Node drain grace period:               %s\n" +
+	"Management upgrade:                    %s\n" +
```

Comment:
Comment as above, if it's optional, we should also reflect in describe. Only display `Management upgrade` if we do have max surge and unavailable

Reply:
It will always have a value, it defaults to maxSurge=1, maxUnavailable=0.